examples/bickleyjet/bickleyjet_cg_invariant_hypervisc.jl

```diff
@@ -14,7 +14,7 @@ import ClimaCore:
 using OrdinaryDiffEq: ODEProblem, solve, SSPRK33
 
 using Logging
-ClimaComms.@import_required_backends
+@static pkgversion(ClimaComms) >= v"0.6" && ClimaComms.@import_required_backends
```

Comment:
I don't know that it's worth adding these conditionals -- to do this properly / thoroughly, we would also need to add similar conditionals around every new feature since ClimaComms version 0.6 (e.g., `ClimaComms.allowscalar` and more), could we just require 0.6 for this version instead? Also, to be sure that it was done correctly, we should probably get #1781 working.

In the same breadth, if this is for some reason a headache for you for testing purposes, I'm fine with temporarily adding it.

Reply:
Okay, then we should bump the compat to `0.6`

Reply:
I removed the conditional and bumped the compat to 0.6